onnx/defs/generator/defs.cc

```diff
@@ -59,10 +59,10 @@ ONNX_OPERATOR_SET_SCHEMA(
         .Attr(
             "dtype",
             "(Optional) The data type for the elements of the output tensor. If not specified,"
-            "the data type of the input tensor T1 is used. If input tensor T1 is also not" 
-            "specified, then type defaults to 'float'.",
+            "the data type of the input tensor T1 is used. If input tensor T1 is also not " 
+            "specified, then output tensor type defaults to 'float'.",
```

Comment:
default type should be the same as the input tensor?

Reply:
Yes, if `dtype` is not specified, then output type should be the same as input tensor. This is the case when even input tensor is not provided (i.e. output created using `shape` attribute).

Reply:
Yeah, I mean you didn't change the description :-)

Reply:
umm, sorry, I am not sure if the description needs to be changed. The lines you are pointing to (62-63) are for the case when both `dtype` and input tensor are not provided, in which case the output tensor type, by default, should be float. 

Maybe I am misunderstanding your comment?

Reply:
Cool, thanks.